core/dbt/adapters/base/impl.py

```diff
@@ -1273,14 +1291,8 @@ def _parse_column_constraint(cls, raw_constraint: Dict[str, Any]) -> ColumnLevel
         except Exception:
             raise DbtValidationError(f"Could not parse constraint: {raw_constraint}")
 
```

Comment:
We were just looking at a single constraint int eh macro but we decided to move all the logic into Python so that we pass in the entire column.  The same logic is handled, just differently.